packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AuthCognito.kt

```diff
@@ -447,6 +445,13 @@ public class AuthCognito : FlutterPlugin, ActivityAware, MethodCallHandler, Plug
     }
   }
 
+  fun prepareFetchAttributeResult(@NonNull flutterResult: Result, @NonNull attributes: List<AuthUserAttribute>) {
```

Comment:
Should it be called prepareFetchAttribute**s**Result

Reply:
Done.